habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -95,6 +97,16 @@ def __init__(
         self._play_episodes_filter_str = self._hitl_config.episodes_filter
         self._num_recorded_episodes = 0
         self._gui_input = gui_input
+        logging.basicConfig(
+            filename="/home/priyamp/hitl/app_timing.log",
```

Comment:
We'll want to change this default path :)

We probably also want to make this logging optional.